sos/report/__init__.py

```diff
@@ -1081,8 +1077,8 @@ def list_plugins(self):
                 if tmpopt is None:
                     tmpopt = 0
 
-                self.ui_log.info(" %-25s %-15s %s" % (
-                    opt.plugin + "." + opt.name, tmpopt, opt.desc))
+                self.ui_log.info(f" {opt.plugin}.{opt.name:<25} {tmpopt:<15} "
```

Comment:
Currently, we apply the "right alignment to 25characters" rule to `{opt.plugin}.{opt.name}`. Newly we will apply it just to `{opt.name}`.

We might have there

    f" {f'{opt.plugin}.{opt.name}':<25} {tmpopt:<15}"

or use a new variable.